packages/block-library/src/reset.scss

```diff
@@ -5,16 +5,17 @@
  * of the editor by themes.
  */
 
+// The following styles revert to the browser defaults overriding the WPAdmin styles.
+// This is only needed while the block editor is not being loaded in an iframe.
```

Comment:
This seems the case for the whole file isn't it?

Reply:
Yes, that's why I moved the comment upwards :) I can rephrase and make it part of the initial statement?